### DISCUSSION TITLE
Feature request: `outdated` command

### DISCUSSION THREAD

**Discussion by jimeh** [jdx/mise#525] [Ideas]
Apologies if I've missed something, but only way I've found to get details about available updates to installed tools is by abusing the `global` command.

In my `~/.tool-versions` file I have all tools set to `latest` (except nodejs which is `lts`), and only way I've found to get rtx to tell me if there are newer versions available is via the global command:

```command
$ rtx global buf@latest
[WARN] Tools not installed: python@3.11.3, terraform@1.4.6
```

I already have `buf` set to `latest`, but running the `global` command again lists outdated tools.

**Comment by jdx** [comment]
Agreed, this would be useful

**Comment by jimeh** [comment]
This was added in [v1.30.0](https://github.com/jdxcode/rtx/releases/tag/v1.30.0), so I’ll close this discussion.

Thanks for your efforts @jdxcode :)
